### PR TITLE
Add unitctl quickstart to README

### PR DESCRIPTION
**Pull request by danielledeleo**
This PR adds quick start instructions for `unitctl`. Some installation steps were also edited for clarity.

Blocked by: nginx/homebrew-unit#39 until a decision is made there. 

I think we can remove the PHP hello world section entirely for brevity.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,45 +61,112 @@ Your current working directory will now be mounted to the Unit image at `/www`.
...
+[unit.nginx.org](https://unit.nginx.org/howto/) for web application frameworks
+built with Python, PHP, WebAssembly, Node.js, Ruby, and more.
 
-NGINX Unit provides a RESTful API for dynamic configuration.
-See the [control API documentation](https://unit.nginx.org/controlapi/)
-for more information on what endpoints are available and how to use them.
+Here's an example using the `unit:python` Docker image:
+```
+unitctl instances new 127.0.0.1:8001 /path/to/app 'unit:python'
+```
```
Comment (ac000) [inline README.md:122]: To be consistent this snippet should be of type `console` and could you put the prompt character in to show who it's intended that the command be run as.

File: README.md [hunk 2]
```diff
@@ -61,45 +61,112 @@ Your current working directory will now be mounted to the Unit image at `/www`.
...
+```python
+def application(environ, start_response):
+    start_response("200 OK", [("Content-Type", "text/plain")])
+    return (b"Hello, Python on Unit!")
+```
+
+You can then interactively edit the currently active configuration:
+``` sh
+unitctl edit
+```
```
Comment (ac000) [inline README.md:136]: Same here... ( with s/sh/console/)

File: README.md [hunk 3]
```diff
@@ -61,45 +61,112 @@ Your current working directory will now be mounted to the Unit image at `/www`.
...
+        "path": "/www/",
+        "module": "wsgi"
+    }
+  },
+  "listeners": {
+    "*:8000": {
+      // Point listener to new application
+      "pass": "applications/python"
+    }
+  }
```
Comment (ac000) [inline README.md:152]: We normally put `listeners` before `applications`

File: README.md [hunk 4]
```diff
@@ -61,45 +61,112 @@ Your current working directory will now be mounted to the Unit image at `/www`.
...
-# ./setup-unit welcome
+# ./setup-unit repo-config
+```
+
+Debian derivatives:
+``` console
+# apt install unit
+```
+
+RHEL derivatives:
```
Comment (ac000) [inline README.md:77]: Technically that's `Fedora derivatives` seeing as Fedora is the upstream for RHEL..

**Review by avahahn — APPROVED** [review]
This looks phenomenal! I am super excited for this to get into the readme!

**Review by ac000 — APPROVED** [review]

**Pull request merged**